show/main.py

```diff
@@ -226,6 +226,25 @@ def sfp(interfacename):
 
     run_command(cmd)
 
+#
+# 'mac' command ("show mac ...")
```

Comment:
>mac [](start = 23, length = 3)

I am not sure about other vendors' CLI. Is it more accurate to call it 'show fdb'?

Reply:
@lguohan , can you suggest?